src/main/scala/wdl4s/UnsatisfiedInputsException.scala

```diff
@@ -3,3 +3,6 @@ package wdl4s
 class UnsatisfiedInputsException(message: String, cause: Throwable) extends RuntimeException(message, cause) {
   def this(message: String) = this(message, null)
 }
+class UnsatisfiedInputsExceptionWithErrors(message: String, cause: Throwable, val errors: Seq[String] = Seq.empty) extends UnsatisfiedInputsException(message, cause) {
```

Comment:
Since you say it's with Errors then errors should be a NEL, there should be no possibility of it being empty